docs/en/admins/07_LinuxUpdate.md

```diff
@@ -1,8 +1,8 @@
 # Updating on Linux
 
-This tutorial demonstrates commands for updating FreshRSS. It assumes that your main FreshRSS directory is `/usr/share/FreshRSS`; If you've installed it somewhere else, substitute your path as necessary.
+This tutorial demonstrates commands for updating FreshRSS. It assumes that your main FreshRSS directory is `/usr/share/FreshRSS`; If you have installed it somewhere else, substitute your path as necessary.
 
-**Note that FreshRSS contains a built-in update system.** It's easier to use if you don't understand the commands that follow. It's available through the web interface of your FreshRSS installation, Administration → Update.
+**Note that FreshRSS contains a built-in update system.** It’s easier to use if you do not understand the commands that follow. It’s available through the web interface of your FreshRSS installation, Administration → Update.
```

Comment:
Well, the formality thing occurs a bit more than just once or twice. :)

I'm sure the newly English comments are fine but I haven't had time to look at them yet.

Reply:
I have reverted the formality fixes